sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/SendEventOptions.cs

```diff
@@ -64,6 +65,17 @@ internal class SendEventOptions
         ///
         public string PartitionId { get; set; }
 
+        /// <summary>
+        ///   If a <see cref="SendEventOptions.PartitionId"/> is specified, it allows to set how long a dedicated transport producer
+        ///   would sit in memory before its pool would remove and close it.
+        /// </summary>
+        ///
+        /// <remarks>
+        ///   If no duration is specified a default value will be used by the pool.
+        /// </remarks>
+        ///
+        public TimeSpan? RemoveAfterDuration { get; set; }
```

Comment:
This isn't something that we want to expose as "when sending" option;  this is an implementation detail to the producer client.   If we see a future need, we'd want to expose this as part of the producer client options, but for now, I'd prefer to go with a private value.